prediction_market_agent_tooling/markets/omen/omen.py

```diff
@@ -595,11 +598,15 @@ def redeem_positions_from_all_omen_markets() -> None:
     Redeems positions from all resolved Omen markets.
     """
     keys = APIKeys()
-    resolved_omen_bets = get_resolved_omen_bets(
+    omen_subgraph_handler = OmenSubgraphHandler()
+    bets = omen_subgraph_handler.get_bets(
```

Comment:
Couldn't we use `get_resolved_bets` here?

```suggestion
    bets = omen_subgraph_handler.get_resolved_bets(
```

Reply:
good catch